webapp/templates/partials/scoreboard_table.html.twig

```diff
@@ -419,9 +588,46 @@
     {% endfor %}
 </style>
 <script>
-    document.querySelectorAll(".forceWidth:not(.toolong)").forEach(el => {
+    document.querySelectorAll(".desktop-scoreboard .forceWidth:not(.toolong)").forEach(el => {
         if (el instanceof Element && el.scrollWidth > el.offsetWidth) {
             el.classList.add("toolong");
         }
     });
+    window.resizeMobileTeamNames = function() {
```

Comment:
should we move this to our JS file so it is not loaded every time?

Reply:
Good idea, should be done now.